test/software/amazon/ion/SymbolTest.java

```diff
@@ -216,7 +200,9 @@ public void testCloneSymbolWithKnownText()
     @Test
     public void testSymbolWithEscapedNewline()
     {
-        badValue("'\\\n'");
```

Comment:
changed it by mistake, added it back again as a valid case. Will take a look if this is included in ion-tests and add it in a separate PR if necessary